bin/functions/workload-functions.sh

```diff
@@ -308,6 +309,17 @@ function ensure-mahout-release (){
     export_withlog HADOOP_CONF_DIR    
 }
 
+function ensure-tpcds-kit-ready (){
+    if [ ! -e "${DSDGEN_DIR}/dsdgen" ]; then
+        mv "${DSDGEN_DIR}/Makefile.suite" "${DSDGEN_DIR}/makefile"
```

Comment:
Seems this can't be done, because the working folder should be Makefile.suite's folder.